packages/unplugin-typia/src/core/index.ts

```diff
@@ -88,8 +88,12 @@ const unpluginFactory: UnpluginFactory<
 				if (next != null && next[0] === 1) {
 					const replaceText = next[1];
 
-					/** get first non-whitespace character of text (maybe bug of magic-string) */
-					const firstNonWhitespaceIndexOfText = text.search(/\S/);
+					/**
+					 * get first non-whitespace character of text (maybe bug of magic-string)
```

Comment:
Additionally, I did not understand why we need to preserve spaces in the replace target.
However, I explicitly added handling for \n, as it could clearly break the code.

Reply:
Thank you for your fix. I think this is reasonable! Thanks